sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderRegistry.java

```diff
@@ -235,7 +235,9 @@ public void registerCoderForType(TypeDescriptor<?> type, Coder<?> coder) {
    *
    * @throws CannotProvideCoderException if a {@link Coder} cannot be provided
    */
+  // CHECKSTYLE.OFF: MissingDeprecated
   @Deprecated
+  // CHECKSTYLE.ON: MissingDeprecated
```

Comment:
If you don't believe that is possible, we should change the `@deprecated` comment to be something like:
`@deprecated This method is to change in an unknown backwards incompatible way once support for this functionality is refined.`

Similarly for the other CoderRegistry `@Internal/@Deprecated` methods.